### PR TITLE
Download Tenant theme feature

### PR DESCRIPTION
**Pull request by kavishkafernando**
This PR includes,

- Fixes https://github.com/wso2/product-apim/issues/8659
- Download Tenant theme feature

Screen Shot 2020-07-13 at 3 07 52 PM
Screen Shot 2020-07-13 at 3 08 07 PM

### CODE REVIEW

File: features/apimgt/org.wso2.carbon.apimgt.admin.feature/src/main/resources/admin/source/src/app/components/TenantTheme/UploadTheme.jsx [hunk 1]
```diff
@@ -94,20 +115,47 @@ const useStyles = makeStyles((theme) => ({
 function UploadTheme() {
     const classes = useStyles();
     const [themeFile, setThemeFile] = useState([]);
+    const [fileName, setFileName] = useState('');
```
Comment (tmkasun) [inline features/apimgt/org.wso2.carbon.apimgt.admin.feature/src/main/resources/admin/source/src/app/components/TenantTheme/UploadTheme.jsx:118]: Do we need to initialize the state with an empty string ?

File: features/apimgt/org.wso2.carbon.apimgt.admin.feature/src/main/resources/admin/source/src/app/components/TenantTheme/UploadTheme.jsx [hunk 2]
```diff
@@ -94,20 +115,47 @@ const useStyles = makeStyles((theme) => ({
...
     const onDrop = (acceptedFile) => {
         setIsFileAccepted(true);
         setThemeFile(acceptedFile[0]);
     };
 
+    useEffect(() => {
+        const tenantThemeContent = restApi.exportTenantTheme();
+        tenantThemeContent.then((response) => {
+            const tenantFileName = response.headers['content-disposition'].split('filename=')[1];
+            setFileName(tenantFileName.slice(1, -1));
```
Comment (tmkasun) [inline features/apimgt/org.wso2.carbon.apimgt.admin.feature/src/main/resources/admin/source/src/app/components/TenantTheme/UploadTheme.jsx:133]: Why do we need to remove the first character?

File: features/apimgt/org.wso2.carbon.apimgt.admin.feature/src/main/resources/admin/source/src/app/components/TenantTheme/UploadTheme.jsx [hunk 3]
```diff
@@ -94,20 +115,47 @@ const useStyles = makeStyles((theme) => ({
...
+    }, []);
+
+    /**
+     * Downloads Tenant Theme ZIP file.
+     *
+     */
+    const handleDownloadTenantTheme = () => {
+        const tenantThemeContent = restApi.exportTenantTheme();
+        tenantThemeContent.then((response) => {
+            Utils.forceDownload(response);
```
Comment (tmkasun) [inline features/apimgt/org.wso2.carbon.apimgt.admin.feature/src/main/resources/admin/source/src/app/components/TenantTheme/UploadTheme.jsx:144]: We can simplify this as 

```
tenantThemeContent.then(Utils.forceDownload)
```

no need to create a lambda/arrow function there

File: features/apimgt/org.wso2.carbon.apimgt.admin.feature/src/main/resources/admin/source/src/app/components/TenantTheme/UploadTheme.jsx (hunk 2 above)
Comment (kavishkafernando) [reply features/apimgt/org.wso2.carbon.apimgt.admin.feature/src/main/resources/admin/source/src/app/components/TenantTheme/UploadTheme.jsx:133]: fileName comes as **filename="test1.com"** so need to remove the "

File: features/apimgt/org.wso2.carbon.apimgt.admin.feature/src/main/resources/admin/source/src/app/components/TenantTheme/UploadTheme.jsx (hunk 3 above)
Comment (kavishkafernando) [reply features/apimgt/org.wso2.carbon.apimgt.admin.feature/src/main/resources/admin/source/src/app/components/TenantTheme/UploadTheme.jsx]: Fixed with `2d41afa`

File: features/apimgt/org.wso2.carbon.apimgt.admin.feature/src/main/resources/admin/source/src/app/components/TenantTheme/UploadTheme.jsx (hunk 1 above)
Comment (kavishkafernando) [reply features/apimgt/org.wso2.carbon.apimgt.admin.feature/src/main/resources/admin/source/src/app/components/TenantTheme/UploadTheme.jsx]: Fixed with `2d41afa`

**Pull request merged**